gems/sorbet-runtime/lib/types/non_forcing_constants.rb

```diff
@@ -4,8 +4,8 @@
 module T::NonForcingConstants
   # NOTE: This method is documented on the RBI in Sorbet's payload, so that it
   # shows up in the hover/completion documentation via LSP.
-  T::Sig::WithoutRuntime.sig {params(val: BasicObject, klass: String).returns(T::Boolean)}
-  def self.non_forcing_is_a?(val, klass)
+  T::Sig::WithoutRuntime.sig {params(val: BasicObject, klass: String, package: T.nilable(String)).returns(T::Boolean)}
+  def self.non_forcing_is_a?(val, klass, package: nil)
```

Comment:
This will need more changes in the future to actually look up the package, but this will work for now for our one-massive-fake-package approach.

Reply:
Could you put in a TODO or a note to this effect?